config/settings.yml

```diff
@@ -69,3 +69,6 @@
         :ems_refresh_worker_amazon_network: {}
         :ems_refresh_worker_amazon_ebs_storage: {}
         :ems_refresh_worker_amazon_s3: {}
+:prototype:
```

Comment:
note that settings will be merged with the core configs...
is there already a top level prototype key?
otherwise I'd put it under`ems -> ems_amazon -> s3_prototype` e.g.